src/components/ProfileDropdown.jsx

```diff
@@ -53,12 +53,9 @@ const useStyles = makeStyles((theme) => ({
     alignSelf: 'center',
   },
   medium: {
-    width: '48px',
-    height: '48px',
-  },
-  large: {
-    width: '60px',
-    height: '60px',
+    width: '50px',
+    height: '50px',
+    marginLeft: theme.spacing(2),
```

Comment:
Is this fixing a spacing issue within the `ProfileDropdown` component? If it isn't then don't add this here. Add the fix in the component where that spacing matters, i.e. in this case `HeaderNavMenu` is where needs to be enough width/spacing between the LoginButton/ProfileDropdown and the other buttons.

Reply:
This is actually for the dropdown avatar but I removed the marginLeft and the sizing is like that because in the previous PR you mentioned the default size of the avatar within the dropdown was too small. These changes no longer correspond to the avatar within the HeaderNavMenu